languages/javascript/exercises/concept/type-conversion/.docs/hints.md

```diff
@@ -0,0 +1,17 @@
+## 1. Calculate the sum of those two array
+
+- There is a built-in function to join the `array` and convert it to a string doing the implicit
+  type conversion to make it string
+- There is built in object which is explicitly used to convert the `string` to the `number`
+- Sum the result of the two converted number
+
+## 2. Calculate the function which tells us the owner whether the number is lucky or not
+
+- There is a prototype which explicit convert the number to `string`
+- There is a function which can convert the `string` to the `array`.
+- There is a function on the `array` constructor which can reverse the number of elements in your array
+- First convert the number into the string and then convert it into array and calculate the reverse of it and join the array and finally compare the two string
+
+## 3. Dashify the number
+
+- Convert the number into string and then convert it into the array and finally join the string using the function to `join` the array using dash
```

Comment:
Try not to give away the solution with the hint.

```suggestion
- Use type conversion to change the number to a string
- Strings have methods which can be used to break them apart
- Arrays have a method to merge elements into a string
```

You could apply this to each hint as well